run-cnf-suites.sh

```diff
@@ -104,6 +104,7 @@ if [[ -z "${TNF_PARTNER_SRC_DIR}" ]]; then
 	echo "env var \"TNF_PARTNER_SRC_DIR\" not set, running the script without updating infra"
 else
 	make -C $TNF_PARTNER_SRC_DIR install-partner-pods
+	make -C $TNF_PARTNER_SRC_DIR install-litmus
```

Comment:
Shouldn't we remove the litmus operator after TNF has finished, after line 128?
```
if [[ ! -z "${TNF_PARTNER_SRC_DIR}" ]]; then
	make -C $TNF_PARTNER_SRC_DIR delete-litmus
fi
```

Reply:
Good suggestion, I'll add that.

Reply:
Added the removal after the call to the test.